.github/data/distros.yml

```diff
@@ -162,6 +162,21 @@ include:
     test:
       ebpf-core: true
 
+  - distro: opensuse
+    version: "tumbleweed"
+    eol_check: false
+    base_image: opensuse/tumbleweed:latest
+    jsonc_removal: |
+      zypper rm -y libjson-c-devel
+    packages:
+      type: rpm
+      repo_distro: opensuse/tumbleweed
+      arches:
+        - x86_64
+        - aarch64
+    test:
+      ebpf-core: true
```

Comment:
@thiagoftsm, do you think that we need ebpf tests for `open suse tumbleweed`

Reply:
I think so, I remember users that had issue in the past with `OpenSUSE 15.4`.